ml_genn/converters/few_spike.py

```diff
@@ -67,4 +67,7 @@ def pre_compile(self, tf_model):
             return PreCompileOutput(max_activations={}, max_input=None)
     
     def post_compile(self, mlg_model):
-        pass
+        # do not allow multiple input or output layers
+        if len(mlg_model.inputs) > 1 or len(mlg_model.outputs) > 1:
```

Comment:
why are either of things problematic for few-spike? without adding synaptic delays (which @tnowotny agrees is the solution), issues are going to occur if the graph contains branching paths with different numbers of ml genn layers which subsequently rejoin (if that makes sense)

Reply:
Eventually, yes. Just wanted to get the basic version out now :)